fat32/sdcard.s

```diff
@@ -329,8 +326,39 @@ sdcard_read_sector:
 	clc
 	rts
 
-.ifdef FAST_READ
-@start:	; Enable auto-tx mode
+@start:
+	bit sd_fast
+	bvs @fast
```

Comment:
minor: since fast is hoped to eventually be the default, perhaps the branch should go to slow and fast should be a cycle faster?

Reply:
The branch would be too far to jump over the fast routine, so nothing would be saved here.